proto/substrait/algebra.proto

```diff
@@ -131,14 +131,42 @@ message ReadRel {
       message ArrowReadOptions {}
       message OrcReadOptions {}
       message DwrfReadOptions {}
+      message DelimeterSeparatedTextReadOptions {
```

Comment:
I think we should add boolean  treat_empty_string_as_null

Reply:
If null_value is empty would that achieve enough of the same result as treating empty string as NULL?  (Note that if we have empty string and quoted empty string it's likely only one would match depending on null_value behavior.)

Reply:
I'm fine using that configuration to represent this assuming we remove the "with quotes" part of it.

Reply:
Fine.  Leaving treat_empty_string_as_null as part of value_treated_as_null.